lms/djangoapps/grades/api/v1/views.py

```diff
@@ -146,9 +146,21 @@ class CourseEnrollmentPagination(CursorPagination):
     """
     Paginates over CourseEnrollment objects.
     """
-    page_size = 50
     ordering = 'id'
+    page_size = 50
+    page_size_query_param = 'page_size'
+
+    def get_page_size(self, request):
+        """
+        Get the page size based on the defined page size parameter if defined.
+        """
+        try:
+            page_size_string = request.query_params[self.page_size_query_param]
+            return int(page_size_string)
+        except (KeyError):
```

Comment:
You might want to catch `ValueError`, too, in case someone makes a call like `?page_size=foo`.

Reply:
Good point. Added `ValueError` now. Thanks